lib/order.rb

```diff
@@ -0,0 +1,45 @@
+class Order
+
+  DEFAULT_QUANTITY = 1
+
+  attr_reader :basket
+
+  def initialize
+    @basket = Hash.new
+  end
+
+  def add(item, quantity = DEFAULT_QUANTITY)
+    quantity = @basket[item] + quantity if @basket[item]
+    @basket[item] = quantity
+  end
+
+  def remove(item)
+    fail 'Item not in basket' if !@basket[item]
+    quantity = @basket[item]
+    if
+      quantity > 1
+    then
```

Comment:
Style/MultilineIfThen: Do not use `then` for multi-line `if`. (https://github.com/bbatsov/ruby-style-guide#no-then)